examples/restaurant_visits/run_without_frameworks_utility_analysis.py

```diff
@@ -139,12 +139,13 @@ def aggregate_utility_analysis():
     data_extractors = get_data_extractors()
     public_partitions = list(range(1, 8)) if FLAGS.public_partitions else None
 
-    options = analysis.utility_analysis.UtilityAnalysisOptions(
-        1, 1e-5, aggregate_params, get_multi_params())
+    options = analysis.UtilityAnalysisOptions(1, 1e-5, aggregate_params,
```

Comment:
nit: I would slightly prefer adding names to the non-obvious arguments if possible (i.e., epsilon = 1.0, delta = 1e-5)